golang/tasks/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Set GOPATH
-  lineinfile: "dest=/home/vagrant/.bash_profile line='export GOPATH=$GOPATH:{{ golang_gopath }}'"
+  lineinfile: "dest={{ lookup('env','HOME') }}/.bash_profile line='export GOPATH=$GOPATH:{{ golang_gopath }}'"
```

Comment:
Here is a place that needs .bash_profile to be present. I think the bento boxes have a bashrc file but not a bash_profile